src/components/sortableGridList/usePresenter.ts

```diff
@@ -0,0 +1,63 @@
+import _ from 'lodash';
+import {Easing, useSharedValue} from 'react-native-reanimated';
+import Constants from '../../commons/Constants';
+
+export const WINDOW_WIDTH = Constants.windowWidth;
+export const DEFAULT_NO_OF_COLUMNS = 3;
+export const getItemSize = (numOfColumns: number, viewWidth: number) => viewWidth / numOfColumns;
+
+export const animationConfig = {
+  easing: Easing.inOut(Easing.ease),
+  duration: 350
+};
+
+export type ItemsLayouts = ({width: number; height: number} | undefined)[];
+export type ItemsOrder = number[];
+
+const usePresenter = (itemsSize: number, numOfColumns: number, itemSpacing: number) => {
```

Comment:
By `itemsSize` do you mean the number of items?

Reply:
true (: 
Think I should rename?

Reply:
Yup :)